integration-tests/tests/sharding.rs

```diff
@@ -278,22 +280,27 @@ async fn invoke_and_await_workers(workers: &[WorkerId]) -> Result<(), worker::wo
 
     for worker_id in workers {
         let worker_id_clone = worker_id.clone();
-        tasks.push(tokio::spawn(async move {
-            let idempotency_key = IdempotencyKey::fresh();
-            DEPS.invoke_and_await_with_key(
-                &worker_id_clone,
-                &idempotency_key,
-                "golem:it/api/echo",
-                vec![Value::Option(Some(Box::new(Value::String(
-                    "Hello".to_string(),
-                ))))],
-            )
-            .await
-        }));
+        tasks.push((
```

Comment:
Fyi: there is a nicer way to spawn and await many subtasks, called `JoinSet` - you `spawn` on the joinset itself and then do a `join_next` loop on it. I started using that in all the benchmarks.

Reply:
thanks! i will change it in the next PR